docs/admin/authorization/rbac.md

```diff
@@ -609,12 +611,24 @@ subjects:
   name: user-1
 ```
 
+### Bootstrapping
+
 When bootstrapping the first roles and role bindings, it is necessary for the initial user to grant permissions they do not yet have.
 To bootstrap initial roles and role bindings:
 
 * Use a credential with the `system:masters` group, which is bound to the `cluster-admin` super-user role by the default bindings.
 * If your API server runs with the insecure port enabled (`--insecure-port`), you can also make API calls via that port, which does not enforce authentication or authorization.
 
+### Pod Creation
```

Comment:
this also isn't specific to RBAC... it applies equally to all authorization methods, so should probably go in the top-level authorization topic

Reply:
Think I should move it to the `index.md` doc?